examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -125,20 +122,32 @@ def _save_forcefield(filename, ff_params):
 
 
 if __name__ == "__main__":
+    parser = ArgumentParser(description="Fit Forcefield parameters to hif2a")
+    parser.add_argument("--num-gpus", default=NUM_GPUS, help="Number of GPUs to run against")
+    parser.add_argument("--hosts", nargs="*", default=None, help="Hosts running GRPC worker to use for compute")
```

Comment:
We should probably indicate and check that --hosts and --num_gpus is an XOR relationship